docs/src/developer/installing.rst

```diff
@@ -0,0 +1,57 @@
+***************
+Getting started
+***************
+In this section, we'll guide you to install the dependencies and environment to develop on Oríon.
+We made our best to automate most of the process using Python's ecosystem goodness to facilitate
+your onboarding. Let us know how we can improve!
+
+Oríon
+=====
+The first step is to clone your remote repository from Github (if not already done, make sure to
+fork our repository_ first).
+
+.. code-block:: sh
+
+   $ git clone https://github.com/epistimio/orion.git --branch develop
+
+.. tip::
+
+   The usage of a `virtual environment`_ is recommended, but not necessary.
+
+   .. code-block:: sh
+
+      $ mkvirtualenv -a $PWD/orion orion
+
+Then, you need to deploy the project in `development mode`_ by invoking the ``setup.py`` script with
+``develop`` argument or by using ``pip install --editable``.
+
+.. code-block:: sh
+
+   $ python setup.py develop --optimize=1
+
+Database
+========
+Follow the same instructions as to install the :ref:`install_database` regularly.
+
+Verifying the installation
+==========================
+For developer's convenience the packages enlisted in the requirements file
+``dev-requirements.txt`` are meant to facilitate the development process.
+Packages include `tox <https://tox.readthedocs.io/en/latest/>`_ for defining
+and organizing macros of sh commands in virtual environments, and packages
+for linting as we will see in a next chapter.
+
+Check everything is ready by running the test suite using ``$ tox`` (this will take some time).
```

Comment:
This will fail because the backward-compatibility tests require some environment variables to be set. I would say `tox -e py36` could be sufficient, otherwise could add most important subset `tox -e flake8,pylint,docs,py36`.

Reply:
Good point. I didn't wanted to include `py36` in case it's not installed on their machine. I'm changing it to your suggestion in the meantime.

Reply:
We can add a note that 37 or 38 or available as well if this is their python version.